voluncheer/opportunityboard/views/opportunityboard.py

```diff
@@ -93,6 +93,8 @@ def deregister_volunteer(request, opportunity_id):
 def organization_view(request, userid):
     organization = Organization.objects.get(pk=userid)
     user = get_object_or_404(User, pk=request.user.pk)
+    if user.is_organization:
```

Comment:
Why do we not allow an organization to see the list of opportunities?

Reply:
Nonono, this is for banning organization viewing volunteer-viewed organization page